build-logic/build-platform/build.gradle.kts

```diff
@@ -16,7 +16,7 @@ val kotlinVersion = providers.gradleProperty("buildKotlinVersion")
 dependencies {
     constraints {
         // Gradle Plugins
-        api("com.gradle:gradle-enterprise-gradle-plugin:3.9")
+        api("com.gradle:gradle-enterprise-gradle-plugin:3.10")
```

Comment:
I think it would also be good to update this location: https://github.com/gradle/gradle/blob/4b4b459880ce85bca733753cabcbe286b3403fb8/build-logic-commons/gradle-plugin/build.gradle.kts#L15

Reply:
Done and updated the plugin release process as well.